lib/tasks/importers.rake

```diff
@@ -21,6 +21,14 @@ namespace :importers do
       MediaImporter.new.run!
     end
   end
+  namespace :viz do
+    require 'viz_import/volume_chapter_importer'
+    desc 'Import Viz Volumes and Chapters'
+    task volume_assoc: :environment do |_t|
+      puts 'Importing Viz Volumes and Chapters'
+      VolumeChapterImporter.new.run!
```

Comment:
Nope, that `initialize` method is for the `ImportFiles` sub class, `VolumeChapterImporter` only has the method `run!`. Had to rebase this anyways for the new mapping stuff